config/default.yml

```diff
@@ -1,6 +1,13 @@
 # =============================================================================
 # Default Runner configuration
 # =============================================================================
+globals:
+    options:
+        working-dir:
+            config: working_dir
+            mode: 2
+            description: Working directory, defaults to current working directory.
+            default: .
 
-taskman:
```

Comment:
@drupol 

This breaks compatibility with phptaskman/drupal that depends on `taskman.bin_dir:` https://github.com/php-taskman/drupal/blob/master/src/Robo/Plugin/Commands/AbstractDrupalCommands.php#L316

Should we release a new version of phptaskman/drupal with a dependency to 1.0.5 or rethink the renaming of this property?

Reply:
We should update it :(

Reply:
Okey, I'll make a pull request.